src/tool/N3ME/DlgBase.cpp

```diff
@@ -834,15 +834,15 @@ BOOL CDlgBase::OnNotify(WPARAM wParam, LPARAM lParam, LRESULT * pResult) {
                 pC->m_Data.fNP = (float)atof(pItem->m_curValue);
             } else if (pItem->m_propName == "Far Plane") {
                 pC->m_Data.fFP = (float)atof(pItem->m_curValue);
-            } else if (pItem->m_propName == "�Ȱ� ���") {
+            } else if (pItem->m_propName == "Use fog") {
```

Comment:
The case here is inconsistent to what you set the properties to.
You called them "use fog", and "fog color" and then you're checking against "Use fog" & "Fog color" -- which won't match because of case sensitivity.

Reply:
Good catch! I also addressed similar places using with breaking changes in terms of properties.